boards/BMS/Src/App/states/App_AllStates.c

```diff
@@ -157,8 +152,7 @@ bool App_AllStatesRunOnTick100Hz(struct StateMachine *const state_machine)
         acc_meas_settle_count++;
     }
 
-    // App_SharedErrorTable_HasAnyCriticalErrorSet left in for now, to be removed entirely in future
-    else if (acc_fault || ts_fault || App_SharedErrorTable_HasAnyCriticalErrorSet(error_table))
+    else if (acc_fault || ts_fault)
     {
         status = false;
         App_SharedStateMachine_SetNextState(state_machine, App_GetFaultState());
```

Comment:
Shouldn't we also check that other boards are not in their fault state?

Reply:
I don't want to accidently regress that we're no longer faulting if some error occurs on another board, could be dangerous so lets be careful here

Reply:
I think the idea is that we want boards (or at least the BMS) to act independently of other boards. @Eddy13013, @joethrstn and I discussed whether there were any boards that could have issues with high voltage being present while in their respective fault states, and couldn't come up with anything. Of course I'll let others chime in if I got any of that wrong.

Reply:
Makes sense. Just wanted to double check there aren't currently any error table errors that we currently rely on to open contactors.